maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -457,9 +443,7 @@ data class ExtractTextWithAICommand(
     override val optional: Boolean = true,
     override val label: String? = null
 ) : Command {
-    override fun description(): String {
-        if (label != null) return label
-
+    override fun originalDescription(): String {
```

Comment:
Won't this break? I didn't imagine to actually change the current description implementation?


I thought we would have description that either resolves to label or evaluate description (display description). On other hand, the original description simply holds the original value of description?

Reply:
Can you write unit test case for these 3 cases:

- original description having the original constructed description
- description evaluating to label if its non null or the description (with **_evaluated env. variable_**). The description also has values that are evaluated in runtime by js engine.

Checkout `YamlCommandReaderTest`, that should help